### ISSUE TITLE
Updating pre-release with Add <Link replace> and <Redirect push>

### ISSUE THREAD

**Issue by PGuimarais** [ReactTraining/react-router#4039]
Was unsure of where the best place to raise this was, but I'm currently waiting on [this merged PR - #3912](https://github.com/ReactTraining/react-router/pull/3912) and I see it was merged into the v4 branch 6 days ago. Was wondering when we were planning on incorporating these changes with the v4.0.0-alpha.x prerelease.

**Comment by timdorr** [comment]
In the next release. That will be soon, I'm sure.
